helm/values.yaml

```diff
@@ -6,13 +6,17 @@ psql:
   job:
     backoffLimit: "0"
     ttlSecondsAfterFinished: "10"
-    activeDeadlineSeconds: "1200"
+    activeDeadlineSeconds: "1800"
     container:
       image:
         repository: "docker-folio.dev.folio.org/org/folio/psql-dump-helm"
         pullPolicy: Always
-        tag: ""
-      envVarsSecret: "s3-postgres-backups-credentials"
+        tag: "1.0.0-RANCHER-313-20220628124331"
+      # The below secret provided with every project deployment and has value of cluster/project name,
+      # aws s3 bucket name, aws s3 access and secret keys
+      envVarsSecret: "aws-postgres-backups-s3-bucket-credentials"
```

Comment:
Should it be "s3CredentialsSecret"? and I guess specific keys must be in comment

Reply:
We took that secret from project rancher terraform deployment secrets list, so, we should give the secret name that could be clear in that context, cause, there is at least 2 another s3 secrets.